RockWeb/Blocks/Finance/ContributionStatementLava.ascx.cs

```diff
@@ -292,6 +292,7 @@ private void DisplayResults()
             qry = qry.OrderByDescending( t => t.Transaction.TransactionDateTime );
 
             var mergeFields = new Dictionary<string, object>();
+            mergeFields.Add( "CurrentPerson", targetPerson);
```

Comment:
I would suggest making this `"Person"` instead of `"CurrentPerson"` since it isn't always the logged in person whose contribution statement is being viewed. You can also pass in a person Guid and it will display that person's contribution statement rather than your own (used by admins). In such a case using `CurrentPerson` would be confusing.